### DISCUSSION TITLE
Can not access GitHub Pages on own domain after moving repository, though I updated all DNS

### DISCUSSION THREAD

**Discussion by kellertuer** [community/community#21780] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>I recently moved one of my repositories</p>
<p><a href="http://github.com/kellertuer/Manopt.jl" rel="nofollow noopener">http://github.com/kellertuer/Manopt.jl</a></p>
<p>to an org</p>
<p><a href="https://github.com/JuliaManifolds/Manopt.jl" rel="nofollow noopener">https://github.com/JuliaManifolds/Manopt.jl</a></p>
<p>and the redirect works fine. Also the documentation (Julia Documenter via GitHub Pages) works fine, as long as one accesses the direct url</p>
<p><a href="http://manoptjl.org/stable/" rel="nofollow noopener">http://manoptjl.org/stable/</a></p>
<p>the base url however</p>
<p><a href="http://manoptjl.org" rel="nofollow noopener">http://manoptjl.org</a></p>
<p>does not work. I checked with my provider, and all A entries are still the same and I updated the CNAME last sunday already from <code>kellertuer.github.io</code> to <code>juliamanifolds.github.io</code> however, the last url posted still opens the first string and can not find a github page there (of course). What did I miss and how can I resolve this?</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey <a class="mention" href="/kellertuer">@kellertuer</a>, fellow Julia user here. My suggestion for this would be to remove the CNAME file from the repo, disable GitHub Pages, and then re-enable it.</p>
